### ISSUE TITLE
Jirb does not find .irbrc on Windows in "My Documents"

### ISSUE THREAD

**Issue by rovf** [jruby/jruby#3675]
https://github.com/jruby/jruby/wiki/GettingStarted#jirb-ruby-interactive-console states:

> If you're on Windows, it goes in your My Documents folder or the folder specified in the HOME environment variable.

I have put a .irbrc file into my "My Documents" folder, containing the following lines:

`
STDOUT.puts "This is the .irbrc in 'My Documents'"
$foo=555
`
I'm invoking jirb as `c:\jruby-1.7.23\bin\jirb.bat`

Neither the text is output, nor is the variable $foo set. I conclude, that my .irbrc has not been loaded.

**Comment by djberg96** [comment]
I think the doc is bad. Just put it in your home directory. The trick is that it ENV['HOME'] needs to be set.

    set HOME=%USERPROFILE% # cmd shell
    $env:HOME = $env:USERPROFILE # powershell

Once that's done (and you've put your irbrc file there) it should work.

**Comment by rovf** [comment]
If I need  to set a particular environment variable for this, wouldn't it maybe a more logical recommendation, to write in the docs that the variable IRBRC should be used? This variable is recognized on every platform. The docs could say - for instance: "If you are on Windows, which does not have the concept of a home directory, set the environment IRBRC to the absolute path of your .irbrc file".

**Comment by djberg96** [comment]
True, it does look for IRBRC first, but either will work. Yes, the docs should be updated.
